From 22b99e3bbb2ca33963e839f12ab95314ab8d9ba9 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Fri, 15 May 2015 16:37:20 +0200
Subject: [PATCH] ppp-manager: fix build with Linux 3.2.0 headers

Fixes build with Ubuntu 12.04.

In file included from ppp-manager/nm-ppp-manager.c:42:0:
/usr/include/linux/if_ppp.h:103:16: error: field 'b' has incomplete type
/usr/include/linux/if_ppp.h:108:21: error: field 'b' has incomplete type

Signed-off-by: Petr Vorel <petr.vorel@gmail.com>
---
 src/ppp-manager/nm-ppp-manager.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ppp-manager/nm-ppp-manager.c b/src/ppp-manager/nm-ppp-manager.c
index e27c82b..d410ebf 100644
--- a/src/ppp-manager/nm-ppp-manager.c
+++ b/src/ppp-manager/nm-ppp-manager.c
@@ -39,6 +39,7 @@
 #ifndef aligned_u64
 #define aligned_u64 unsigned long long __attribute__((aligned(8)))
 #endif
+#include <linux/if.h>
 #include <linux/if_ppp.h>

 #include "NetworkManagerUtils.h"
--
2.1.4
